tests/test_script_queue.py

```diff
@@ -348,6 +349,126 @@ def make_add_data(location, locationSalIndex=0):
         with self.assertRaises(salobj.AckError):
             await self.remote.cmd_showScript.set_start(salIndex=3579, timeout=STD_TIMEOUT)
 
+    async def check_add_log_level(self, log_level):
+        """Test script log level when adding a script to the script queue."""
+        await self.assert_next_queue(enabled=False, running=True)
+
+        await self.remote.cmd_enable.start(timeout=STD_TIMEOUT)
+        await self.assert_next_queue(enabled=True, running=True)
+
+        # pause the queue so we know what to expect of queue state
+        await self.remote.cmd_pause.start(timeout=START_TIMEOUT)
+        await self.assert_next_queue(running=False)
+
+        async with salobj.Remote(domain=self.queue.domain, name="Script", index=I0) as script_remote:
+            await self.remote.cmd_add.set_start(
+                logLevel=log_level,
+                isStandard=False,
+                path="script1",
+                config="",
+                location=Location.LAST,
+                descr="test_add_log_level",
+                timeout=START_TIMEOUT)
+            await self.assert_next_queue(running=False, salIndices=[I0])
+
+            script_state = await script_remote.evt_state.next(flush=False, timeout=START_TIMEOUT)
+            self.assertEqual(script_state.state, ScriptState.UNCONFIGURED)
+            script_state = await script_remote.evt_state.next(flush=False, timeout=STD_TIMEOUT)
+            self.assertEqual(script_state.state, ScriptState.CONFIGURED)
```

Comment:
Please add a comment about why these two states are what they are. You didn't do anything between them, so why did the state change?

Reply:
Also, is all of the stuff above this part of the initial setup? You don't do any testing of log level before now.

Reply:
This is standard script queue behavior. When a script is added to the queue the queue launches it and then configures it. It then outputs its log level. I'm reluctant to add much verbiage to all the tests when this is the standard behavior which I have tried to document in the main description of what scripts are and how the queue runs them.